bookwyrm/templates/book/book.html

```diff
@@ -405,4 +405,7 @@ <h2 class="title is-5">{% trans "Lists" %}</h2>
 {% block scripts %}
 <script src="{% static "js/tabs.js" %}?v={{ js_cache }}"></script>
 <script src="{% static "js/autocomplete.js" %}?v={{ js_cache }}"></script>
+{% if user.show_guided_tour %}
```

Comment:
it's only needed if the user wants the guided tour?

Reply:
am I misreading this? My impression is that it's checking if the logged in user has the guided tour enabled, in which case `request.user` represents the logged in user, rather than `user`, which may or may not be set to anything

Reply:
oh you're right, I misunderstood what you meant! Nice catch.